node/src/components/block_executor.rs

```diff
@@ -83,7 +84,11 @@ type BlockHeight = u64;
 /// The Block executor component.
 #[derive(DataSize, Debug, Default)]
 pub(crate) struct BlockExecutor {
-    genesis_state_root_hash: Option<Digest>,
+    /// Height of the highest known block
```

Comment:
Isn't this just the height of the highest block found in storage at startup?
Without context, I would read this either as the current highest finalized or executed block, but it doesn't seem to get updated at all?

Reply:
Should it be called `initial_block_height` instead?